Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,14 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+function task1(){
+  let result=hogwarts.filter(ele=>ele.house==="Gryffindor")
+  return result.map(ele=>ele.firstName+" "+ele.lastName)
+}
+function task2(){
+  let result=hogwarts.filter(ele=>ele.pet!==null&&ele.occupation=="Teacher"&&ele.house=="Gryffindor")
+  return result.map(({firstName,lastName})=>firstName+" "+lastName)
```

Comment:
This works, but you're only using object destructuring in the `map` callback, not the `filter` one - can you use object destructuring in both?

Reply:
sure thanks! done